src/modules/shared/components/ReadingRoomCardControls/ReadingRoomCardControls.tsx

```diff
@@ -30,16 +29,14 @@ const ReadingRoomCardControls: FC<ReadingRoomCardProps> = ({
 			<div className={styles['c-reading-room-card-controls__label']}>
 				<Icon className={styles['c-reading-room-card-controls__label-icon']} name={icon} />
 
-				<TruncateMarkup lines={2}>
-					<p
-						className={clsx(
-							styles['c-reading-room-card-controls__label-text'],
-							flat && 'u-text-ellipsis'
-						)}
-					>
-						{text}
-					</p>
-				</TruncateMarkup>
+				<p
+					className={clsx(
+						styles['c-reading-room-card-controls__label-text'],
+						flat && 'u-text-ellipsis'
+					)}
+				>
+					{text}
+				</p>
```

Comment:
`TruncateMarkup` is hier bewust toegevoegd in overleg met design, 2 lijnen by default en single-line (`u-text-ellipsis`) indien het een "flat" weergave heeft.

Reply:
Change reverted 👍